play.pokemonshowdown.com/src/panel-battle.tsx

```diff
@@ -387,6 +387,7 @@ class BattlePanel extends PSRoomPanel<BattleRoom> {
 	renderControls() {
 		const room = this.props.room;
 		if (!room.battle) return null;
+		if (room.battle.ended) return this.renderAfterBattleControls(null);
```

Comment:
```suggestion
		if (room.battle.ended) return this.renderAfterBattleControls();
```

No need to pass `request`; you're not using it